dotCMS/src/main/resources/com/dotcms/ai/prompts/default-vision-prompt.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Nice

Reply:
We have to develop something to handle this thing in a standard way (dotCMS way) such as dotConnect or so, otherwise we will have N ways to do the same in a couple years